cumulusci/tasks/push/tests/test_push_tasks.py

```diff
@@ -0,0 +1,481 @@
+    task.sf.query_all.return_value = PACKAGE_OBJS
+    assert task._get_orgs() == ["bar", NAME]
+
+
+def test_schedule_push_org_list_run_task_with_time():
+    query = "SELECT Id, PackagePushRequestId, SubscriberOrganizationKey, Status FROM PackagePushJob WHERE Id = '0DV1R000000k9dEWAQ'"
+    with open(ORG_FILE, "w") as file:
+        file.write(f"{ORG_FILE_TEXT}")
+    task = create_task(
+        SchedulePushOrgList,
+        options={
+            "orgs": ORG_FILE,
+            "version": VERSION,
+            "start_time": "now",
+            "namespace": NAMESPACE,
+        },
+    )
+    task.push = mock.MagicMock()
+    task.sf = mock.MagicMock()
+    task.sf.query_all.return_value = PACKAGE_OBJS
+    task.push.create_push_request.return_value = (task.sf.query_all.return_value, 2)
+    task._run_task()
+    task.sf.query_all.assert_called_with(query)
+    os.remove(ORG_FILE)
+
+
+def test_schedule_push_org_list_run_task_without_time():
+    with open(ORG_FILE, "w") as file:
+        file.write(f"{ORG_FILE_TEXT}")
+    task = create_task(
+        SchedulePushOrgList,
+        options={"orgs": ORG_FILE, "version": VERSION, "namespace": NAMESPACE},
+    )
+    task.push = mock.MagicMock()
+    task.sf = mock.MagicMock()
+    task.sf.query_all.return_value = PACKAGE_OBJS
+    task.push.create_push_request.return_value = (task.sf.query_all.return_value, 2)
+    task._run_task()
+    task.push.create_push_request.assert_called_once()
+    os.remove(ORG_FILE)
+
+
+def test_schedule_push_org_list_run_task_without_orgs():
+    with open(ORG_FILE, "w") as file:
+        file.write(f"{ORG_FILE_TEXT}")
+    task = create_task(
+        SchedulePushOrgList,
+        options={
+            "orgs": ORG_FILE,
+            "version": VERSION,
+            "namespace": NAMESPACE,
+            "start_time": "now",
+        },
+    )
+    task.push = mock.MagicMock()
+    task.push_request = mock.MagicMock()
+    task.sf = mock.MagicMock()
+    task.sf.query_all.return_value = PACKAGE_OBJS
+    task.push.create_push_request.return_value = (task.sf.query_all.return_value, 0)
+    task._run_task()
+    task.push.create_push_request.assert_called_once()
+    os.remove(ORG_FILE)
+
+
+def test_schedule_push_org_list_run_task_many_orgs():
+    query = "SELECT Id, PackagePushRequestId, SubscriberOrganizationKey, Status FROM PackagePushJob WHERE Id = '0DV1R000000k9dEWAQ'"
+    with open(ORG_FILE, "w") as file:
+        file.write(f"{ORG_FILE_TEXT}")
+    task = create_task(
+        SchedulePushOrgList,
+        options={
+            "orgs": ORG_FILE,
+            "version": VERSION,
+            "namespace": NAMESPACE,
+            "start_time": "now",
+        },
+    )
+    task.push = mock.MagicMock()
+    task.sf = mock.MagicMock()
+    task.sf.query_all.return_value = PACKAGE_OBJS
+    task.push.create_push_request.return_value = (task.sf.query_all.return_value, 1001)
+    task._run_task()
+    task.sf.query_all.assert_called_with(query)
```

Comment:
Would be good to also assert that create_push_request was called with the expected arguments.

Reply:
@jofsky I unresolved this one; doesn't look like it was done.

Reply:
@davisagli  I think I now addressed lmk.